zipline/resources/market_data/SPY_benchmark.csv

```diff
@@ -5097,1030 +5097,1261 @@
 2013-04-26 00:00:00+00:00,-0.001766338632349207
 2013-04-29 00:00:00+00:00,0.006698685540950544
 2013-04-30 00:00:00+00:00,0.002385436283741438
-2013-05-01 00:00:00+00:00,-0.008767535070140275
-2013-05-02 00:00:00+00:00,0.009287338893100738
-2013-05-03 00:00:00+00:00,0.010140845070422566
-2013-05-06 00:00:00+00:00,0.0025407448720331516
-2013-05-07 00:00:00+00:00,0.005068611694894365
-2013-05-08 00:00:00+00:00,0.004551045510455154
-2013-05-09 00:00:00+00:00,-0.0028162115832007784
-2013-05-10 00:00:00+00:00,0.0032539292730844327
-2013-05-13 00:00:00+00:00,0.0007955449482894839
-2013-05-14 00:00:00+00:00,0.010333863275039823
-2013-05-15 00:00:00+00:00,0.0053864310355262734
-2013-05-16 00:00:00+00:00,-0.004695400915001202
-2013-05-17 00:00:00+00:00,0.009677029152050265
-2013-05-20 00:00:00+00:00,-5.9901761111702534e-05
-2013-05-21 00:00:00+00:00,0.001437728389145132
-2013-05-22 00:00:00+00:00,-0.007417598851468421
-2013-05-23 00:00:00+00:00,-0.002892786114626733
-2013-05-24 00:00:00+00:00,-0.0008461770927772339
-2013-05-28 00:00:00+00:00,0.005988748412074418
-2013-05-29 00:00:00+00:00,-0.006494287432351253
-2013-05-30 00:00:00+00:00,0.003692046967679552
-2013-05-31 00:00:00+00:00,-0.01441234999698493
-2013-06-03 00:00:00+00:00,0.005567792462065624
-2013-06-04 00:00:00+00:00,-0.004806814724672925
-2013-06-05 00:00:00+00:00,-0.014000978234287031
-2013-06-06 00:00:00+00:00,0.00905314069572749
-2013-06-07 00:00:00+00:00,0.01272045719904158
+2013-05-01 00:00:00+00:00,-0.008768026837593923
+2013-05-02 00:00:00+00:00,0.009287480470740572
```

Comment:
Why the changes here?

Reply:
I needed to rebuild the example data for use in test_examples with the new pandas version. That required downloading benchmarks, but our current benchmarks downloading process doesn't go back as far as the current data. I merged in data here from `resources/market_data_SPY_benchmark.csv`, which had a different range of dates, but also some floating point differences where they overlap (such as here). Another option might be to change the start/end dates of the test, but I was hoping to avoid that.